external/kafka-0-10-sql/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -1 +1 @@
-org.apache.spark.sql.kafka010.KafkaSourceProvider
+org.apache.spark.sql.kafka010.KafkaProvider
```

Comment:
Hi @tcondie, I just happened to look at this PR. I just wonder if this breaks existing codes that use `.format("org.apache.spark.sql.kafka010.KafkaSourceProvider")` although almost no users use this by that name.